components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -85,15 +91,30 @@ export class ServicesSidebarComponent implements OnInit, OnDestroy {
     this.currentPage = 1;
     this.selectedHealth = this.serviceGroupsFacade
       .updateHealthFilter(health, 'applicationsStatusFilter');
-    this.serviceGroupsFacade
-      .updateServicesFilters(health, this.currentPage);
+    this.refresh();
   }
 
   public updatePageNumber(pageNumber: number) {
     this.currentPage = pageNumber;
     this.serviceGroupsFacade
       .updatePageNumber(pageNumber, this.totalServices, this.currentPage, 'applicationsPageChange');
-    this.serviceGroupsFacade
-      .updateServicesFilters(this.selectedHealth, this.currentPage);
+    this.refresh();
   }
+
+  refresh() {
+    if (this.serviceGroupsId) {
+
+      const paramsForDispatch = {
```

Comment:
Strong-typing makes life easier. 😁 
```suggestion
      const paramsForDispatch: GroupServicesFilters = {
```